packages/itwinui-react/src/core/utils/components/LineClamp.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import type { PolymorphicForwardRefComponent } from '../props.js';
+import { Box } from './Box.js';
+import { ShadowRoot } from './ShadowRoot.js';
+
+/** @private */
+export const LineClamp = React.forwardRef((props, forwardedRef) => {
+  const { lines, children, ...rest } = props;
+  return (
+    <Box
+      ref={forwardedRef}
+      {...rest}
+      style={
+        { '--_iui-line-clamp': lines, ...props.style } as React.CSSProperties
+      }
+    >
+      <ShadowRoot css={css}>
+        <slot />
+      </ShadowRoot>
+      {children}
+    </Box>
+  );
+}) as PolymorphicForwardRefComponent<'div', { lines?: number }>;
+
+const css = /* css */ `
+  :host {
```

Comment:
Was `text-overflow: ellipsis;` from the `iui-line-clamp` mixin intentionally left out?

Reply:
I read the [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp) for `-webkit-line-clamp` and learned that the ellipses are automatically added. In that case, can we remove `text-overflow: ellipsis;` from the `iui-line-clamp` mixin?